pkg/api/avatar/avatar.go

```diff
@@ -134,7 +134,7 @@ func newNotFound() *Avatar {
 	// load user_profile png into buffer
 	path := filepath.Join(setting.StaticRootPath, "img", "user_profile.png")
 
-	if data, err := ioutil.ReadFile(path); err != nil {
+	if data, err := ioutil.ReadFile(filepath.Clean(path)); err != nil {
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around `setting.StaticRootPath` should be resolved at configuration loading time.